tournament/scripts/demo/client/submit.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash -ex
+
+# Parameters:
+# $1 = hostname (and optional port)
+# $2 = path to scenario file
+# $3 = path to solution file
+
+HOST=$1
+SCENARIO_FILEPATH=$2
+SOLUTION_FILEPATH=$3
+
+# Example:
+#   tournament/scripts/demo/client/submit.sh localhost:8008 data/scenarios/Challenges/dimsum.yaml data/scenarios/Challenges/_dimsum/solution.sw
```

Comment:
The first time I tried this script it didn't work.  After poking at it for a while I realized the reason is that the scenario file and solution file paths must be either absolute paths, or relative to the repository root.  Would be good to add a note here to that effect.

Reply:
Added a note.